.travis.yml

```diff
@@ -16,6 +16,10 @@ before_script:
 script:
     - ./travis_build_test.sh
 before_deploy:
+    - wget http://repo.continuum.io/miniconda/Miniconda3-4.3.21-Linux-x86_64.sh -O miniconda.sh
+    - bash miniconda.sh -b -p $HOME/miniconda
+    - export PATH="$HOME/miniconda/bin:$PATH"
+    - conda install --yes python=3.6
```

Comment:
Just out of curiosity, what do we need conda for on TravisCI?

Reply:
Some package was complaining about an out-of-date python. Conda gives us a nice python3 environment where sphinx can install its dependencies.